lms/static/js/instructor_dashboard/util.js

```diff
@@ -379,7 +379,7 @@
                 url: listEndpoint,
                 success: function(data) {
                     if (data.tasks.length) {
-                        ths.createTaskListTable(ths.$table_running_tasks, data.tasks);
+                        createTaskListTable(ths.$table_running_tasks, data.tasks);
```

Comment:
You're sure this isn't supposed to be `this.createTaskListTable`? That's what it's referenced as on line 529.